starfish/test/test_config.py

```diff
@@ -82,7 +82,7 @@ def test_starfish_config_value_default_key(monkeypatch):
 
 @mark.parametrize("name,config", (
     ("enabled", {
-        "expected": (2841272, 4e6),
+        "expected": (2658848, 4e6),
```

Comment:
What's going on here?

Reply:
This number was empirically determined, but incorrectly includes the data from sqlite cache files.  2658848 is the actual number of bytes for tile data.